pkg/system/stat_netbsd.go

```diff
@@ -0,0 +1,13 @@
+package system // import "github.com/docker/docker/pkg/system"
+
+import "syscall"
+
+// fromStatT converts a syscall.Stat_t type to a system.Stat_t type
+func fromStatT(s *syscall.Stat_t) (*StatT, error) {
+	return &StatT{size: s.Size,
+		mode: uint32(s.Mode),
+		uid:  s.Uid,
+		gid:  s.Gid,
+		rdev: uint64(s.Rdev),
```

Comment:
Same question here

Reply:
Replying to both questions here: yeah, both conversions should be useless and can be removed (I've also locally patched them and double-checked them).

@thaJeztah OK to open a new PR to remove them?

Thank you very much @viandc, @AkihiroSuda and @thaJeztah for the quick reviews and merge!

Reply:
If you have time to look at cleaning up the duplicates (I think if we keep the conversion in that one to take different cases into account, that's ok)

Reply:
Thanks @thaJeztah, I've just opened #41448 that does that.